.github/DEVELOP.md

```diff
@@ -0,0 +1,54 @@
+# Develop
+
+This guide describes how to setup a local development to work on Moai.
+
+## Before you start
+
+This guide is mainly for contributing code to Moai. For other types of contribution, see the [CONTRIBUTING](./CONTRIBUTING.md) guide.
+
+This is a guide to develop Moai itself. For guides to use Moai, e.g. to create your web apps with it, see [docs.moaijs.com](https://docs.moaijs.com).
+
+To understand the principles that drive the design and development of Moai, see the [Principles](https://docs.moaijs.com/?path=/docs/intro-principles--page) guide.
+
+## Codebase overview
+
+| Path        | Project           | Framework   |
+| ----------- | ----------------- | ----------- |
+| lib/core    | [@moai/core]      | [Rollup]    |
+| lib/gallery | [@moai/gallery]   | [Rollup]    |
+| lib/docs    | [docs.moaijs.com] | [Storybook] |
+| site        | [moaijs.com]      | [Next.js]   |
+
+## Local Development
+
+To have a local development environment, go to the "lib/docs" folder and run `yarn start`:
+
+```
+cd lib/docs
+yarn start
+```
+
+Note that you don't need to run `yarn` first. We intentionally prefixed `yarn start` with `yarn` to ensure you always work on the correct dependencies.
```

Comment:
I think we should specify version of environment and framework tools to develop. E.g: version of node, yarn,... to ensure contributor setup correctly.

Reply:
I don't think it would help much, since it works with both yarn 1 and yarn 2, and compatible with the last 2 LTS version of nodejs

Reply:
But what if contributor use an old version of node that cannot load some of the dependencies. At least we should specify the most lowest version that can work well.
I got the a situation like that when working with vim.md, there were changes that required me to upgrade node to v14 to run while I was trying with node v12.

Reply:
oh that's bad. Ok I'll put the node and yarn version :D